packages/jam/jam-host-calls/accumulate/bless.ts

```diff
@@ -101,11 +101,11 @@ export class Bless implements HostCallHandler {
       authorizers,
       delegator,
       registrar,
-      autoAccumulateEntries,
+      autoAccumulate,
     );
 
     if (updateResult.isOk) {
-      logger.trace`BLESS(${manager}, ${authorizers}, ${delegator}, ${registrar}, ${autoAccumulateEntries}) <- OK`;
+      logger.trace`BLESS(${manager}, ${authorizers}, ${delegator}, ${registrar}, ${autoAccumulate}) <- OK`;
```

Comment:
`Map` does not have a very good `toString` implementation (returns `[object Map]`). Maybe let's do `Array.from(autoAccumulate.entries())` instead?

As a pretty wild idea, we could actually detect the `Map` type in the logger implementation and do this in a lazy fashion only when the log line is actually turned into a string.

Alternatively we can wrap into our `MapWithDebug` (in collections) type and have custom `toString`? We could also create couple of static helper methods to construct the map instead of using the `Map` constructor directly (which supports some crazy shit).